topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
+Use a combo-box if:
```

Comment:
There are several usages of "combo-box" with hyphen in this article, please remove the hyphen everywhere

Reply:
Should it be `Combobox` or `Combo box`?